api/v1beta1/spannerautoscaler_types.go

```diff
@@ -105,6 +105,11 @@ type ScaleConfig struct {
 	// +kubebuilder:validation:MultipleOf=1000
 	ScaledownStepSize int `json:"scaledownStepSize,omitempty"`
 
+	// How often(second) autoscaler is reevaluated for scale down
```

Comment:
> The cool down period (in seconds) between two consecutive scaledown operations

But maybe we should switch this to minutes instead of seconds? We don't really need second level precision here I think.

Reply:
IMO, `ScaledownIntervalInSeconds` or `ScaledownIntervalInMinutes` is less error prone and using duration type (like [`metav1.Duration`](https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Duration)?) is better than it.

Reply:
Yes, I agree. Using "duration" type is better, because that way users can choose their own units like `10m` or `2h`.

Reply:
Thank you!
Fixed https://github.com/mercari/spanner-autoscaler/pull/89/commits/3551d19977e190ca67823ec198ec06ef18323a9c

Reply:
What do you think about this as the helper comment?
> The cool down period between two consecutive scaledown operations. If this option is omitted, the value of the `--scaledown-interval` command line option is taken as the default value.

I think adding the information about default value is important here.

Reply:
I think that it is very good 👍 
I will add it.